softlayer/vm/interface.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 type VMCloudProperties struct {
+	VmNamePrefix             string                               `json:"vmNamePrefix,omitempty"`
```

Comment:
@karafelix 
Hi Kara, 'VmNamePrefix' should be kept due to bluemix usage, thanks!